src/test/java/com/github/checkstyle/regression/internal/GitUtils.java

```diff
@@ -75,8 +80,23 @@ public static File addAnEmptyFileAndCommit(Repository repository, String fileNam
     public static void addAllAndCommit(Repository repository, String message)
             throws GitAPIException {
         try (Git git = new Git(repository)) {
+            final Status status = git.status().call();
```

Comment:
Add a comment before the new code stating this, **but first**,

I commented out this code and got no test failures on Windows. I can't check Linux right now.
Please confirm you get a failure without it.